zerver/openapi/zulip.yaml

```diff
@@ -2939,12 +2939,16 @@ paths:
                                       type: boolean
                                       description: |
                                         Whether an invitation is required to join this organization.
-                                    invite_by_admins_only:
-                                      type: boolean
+                                    invite_to_realm_policy:
+                                      type: integer
                                       description: |
-                                        If true, only organization administrators can invite new users to join.
-                                        If false, any [Member](/help/roles-and-permissions) can invite new users
-                                        to join the organization.  [Help center article](/help/invite-new-users#change-who-can-send-invitations).
+                                        Policy for [who can invite new users](/help/invite-new-users#change-who-can-send-invitations)
+                                        to join the organization:
+
+                                        * 1 = Members only
+                                        * 2 = Administrators only
+
+                                        **Changes**: New in Zulip 4.0 (feature level 47).
```

Comment:
We should add here what the old field was.  E.g. add a new sentence.  "Previously, this was encoded as the boolean field `invite_by_admins_only`".